internal/engine/build_and_deployer.go

```diff
@@ -68,9 +68,9 @@ func (composite *CompositeBuildAndDeployer) BuildAndDeploy(ctx context.Context,
 			return store.BuildResultSet{}, err
 		}
 
-		if _, ok := err.(RedirectToNextBuilder); ok {
-			logger.Get(ctx).Debugf("(expected error) falling back to next build and deploy method "+
-				"after error: %v", err)
+		if redirectErr, ok := err.(RedirectToNextBuilder); ok {
+			s := fmt.Sprintf("falling back to next update method because: %v\n", err)
```

Comment:
is this newline intentional?

Reply:
No; fixed; thanks.

Reply:
haaa it WAS intentional, actually:
![Screen Shot 2019-04-16 at 2 24 25 PM](https://user-images.githubusercontent.com/6332648/56234495-68e68280-6053-11e9-9662-1e1c60380588.png)
(the `write` method doesn't append a newline)